packages/block-editor/src/components/block-switcher/style.scss

```diff
@@ -69,47 +69,73 @@
 // We double the max-width for it to fit both the preview & content but we keep the min width the same for the border to work.
 .components-popover.block-editor-block-switcher__popover .components-popover__content {
 	min-width: 300px;
-	max-width: calc(340px * 2);
 	display: flex;
 	background: $white;
 	padding: 0;
+	overflow: visible;
```

Comment:
What does this do?

Reply:
This is needed as the Preview changed design implementation, so as to be shown outside the lists Popup.

Reply:
This would no longer be needed once we switched to using Popover, correct?

Reply:
Correct.